--- crfsuite/include/os.h	2016-04-10 03:12:45.000000000 -0300
+++ crfsuite/include/os.h.orig	2017-03-23 23:36:48.000000000 -0300
@@ -46,8 +46,11 @@
 #define alloca      _alloca
 #define strdup      _strdup
 #define open        _open
+
+#if        _MSC_VER < 1900
 #define isfinite    _finite
 #define snprintf    _snprintf
+#endif/*_MSC_VER < 1900*/
 
 #ifndef    __cplusplus
 /* Microsoft Visual C specific */
